runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -106,10 +106,10 @@ impl Bank {
             stake_rewards_by_partition,
             old_vote_balance_and_staked,
             validator_rewards,
-            validator_rate,
-            foundation_rate,
-            prev_epoch_duration_in_years,
-            capitalization,
+            validator_rate: _,
```

Comment:
Why is this needed? For types?

Reply:
That is ignored variables. We can put :_ to ignore them as their unused in the function. Simply it is to avoid complains from cargo clippy

Reply:
These variables are not used. They use interesting property of Rust, they return a struct and instead of creating an object corresponding to that struct they declare its fields. I used ":_" in order to avoid cargo clippy complains about unused variables.

for instance
```
warning: unused variable: `validator_rate`
   --> runtime/src/bank/partitioned_epoch_rewards/calculation.rs:109:13
    |
109 |             validator_rate,
    |             ^^^^^^^^^^^^^^ help: try ignoring the field: `validator_rate: _`
    |
    = note: `#[warn(unused_variables)]` on by default
    ```

Reply:
Got it, thanks for the context.